bodo/libs/_puffin.cpp

```diff
@@ -562,24 +564,41 @@ PyObject *get_statistics_file_metadata(
 }
 
 /**
- * @brief Generate an equivalent empty bodo_iceberg_connector.StatisticsFile
+ * @brief Generate an equivalent empty pyiceberg.table.statistics.StatisticsFile
  * object for when the rank is not 0. This is used for type stability when
  * interacting with object mode.
  *
  * @return PyObject*
  */
 PyObject *get_empty_statistics_file_metadata() {
-    PyObject *bic = PyImport_ImportModule("bodo_iceberg_connector");
-    CHECK(bic, "importing bodo_iceberg_connector module failed");
+    PyObject *ice = PyImport_ImportModule("pyiceberg.table.statistics");
```

Comment:
Gap from PyIceberg upgrade. Unsure why this only appeared in the Arrow upgrade though and only in Polaris tests?